README.md

```diff
@@ -166,6 +166,20 @@ $statsd->event('Now it is fixed.',
 - Logging on communication errors with Datadog (uses cURL for API request)
 - Logs via error_log and try/catch block to not throw warnings/errors on communication issues with API
 
+### Origin detection over UDP
```

Comment:
let's point out that it is k8s only for now, otherwise the code snippet is confusing.